test/addons/null-buffer-neuter/test.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+// Flags: --expose-gc
```

Comment:
Yes.  Without it, `v8::Isolate::RequestGarbageCollectionForTesting()` aborts with a run-time error.